src/Orleans.EventSourcing/LogStorage/LogConsistencyProvider.cs

```diff
@@ -38,15 +41,6 @@ public bool UsesStorageProvider
             }
         }
 
-        private static int counter; // used for constructing a unique id
-        private int id;
-
-        /// <inheritdoc/>
-        protected virtual string GetLoggerName()
-        {
-            return string.Format("LogViews.{0}.{1}", GetType().Name, id);
```

Comment:
If you are changing this, there were some tests filtering on the LogViews prefix when configuring logging, that should be updated